surface/include/pcl/surface/impl/mls.hpp

```diff
@@ -396,9 +407,6 @@ pcl::MovingLeastSquares<PointInT, PointOutT>::computeMLSPointNormal (int index,
     case (VOXEL_GRID_DILATION):
     case (DISTINCT_CLOUD):
     {
-      // Take all point pairs and sample space between them in a grid-fashion
-      // \note consider only point pairs with increasing indices
-      mls_result = MLSResult (point, plane_normal, u_axis, v_axis, c_vec, static_cast<int> (nn_indices.size ()), curvature);
```

Comment:
I think it was to suppress warning messages. I can remove the two and default instead?